apps/barnepensjon-ui/src/locales/nn.ts

```diff
@@ -10,6 +10,9 @@ const common = {
     lastName: 'Etternamn',
     name: 'Namn',
     fnrDnr: 'Fødselsnummer / d-nummer',
+    fnrIsUnknown: 'Eg veit ikkje fødselsnummeret. (valfritt)',
+    fnrIsUnknownHelpText: 'Ettersom du ikke oppgir fødselsnummer, kan behandlingen av søknaden ta lengre tid.',
```

Comment:
Denne er kanskje ikke oversatt?

```suggestion
    fnrIsUnknownHelpText: 'Ettersom du ikkje oppgir fødselsnummer, kan behandlingen av søknaden ta lengre tid.',
```